cephadm.yml

```diff
@@ -84,8 +84,10 @@
   - name: disable stderr logging of cluster log
     shell: ceph config set global mon_cluster_log_to_stderr false
 
+  - name: distribute ceph.conf by default
+    shell: ceph config set mgr mgr/cephadm/manage_etc_ceph_ceph_conf true
```

Comment:
@sebastian-philipp  this doesn't seem to work. I tried this with master and pacific. Nothing gets distributed to the clients. Clients are labeled with "_no_schedule" and "clients".